src/MemObject.h

```diff
@@ -54,11 +54,48 @@ class MemObject
 
     void write(const StoreIOBuffer &buf);
     void unlinkRequest() { request = nullptr; }
-    const HttpReplyPointer &getReply() const { return reply_; }
-    void replaceReply(const HttpReplyPointer &r) { reply_ = r; }
+
+    /// HTTP response before 304 (Not Modified) updates
+    /// starts "empty"; modified via replaceBaseReply() or adjustableBaseReply()
+    const HttpReply &baseReply() const { return *reply_; }
+
+    /// \returns nil -- if no 304 updates since replaceBaseReply()
+    /// \returns combination of baseReply() and 304 updates -- after updates
+    const HttpReplyPointer &updatedReply() const { return updatedReply_; }
+
+    /// \returns the updated-by-304(s) response (if it exists)
+    /// \returns baseReply() (otherwise)
+    /// Requires mem_obj.
```

Comment:
Looks like a stale comment: if we cannot call freshestReply() without the object.

Reply:
Bad copy-paste. Please fix:
```suggestion
```